skills/image-explore/generate.py

```
@@ -150,20 +151,22 @@ def generate_one(
     env["ASPECT_RATIO"] = aspect
 
     print(f"Generating: {output}", file=sys.stderr)
+    t0 = time.monotonic()
     result = subprocess.run(cmd, env=env, capture_output=True, text=True)
```

Ruff (0.15.2)

[error] `subprocess` call: check for execution of untrusted input

(S603)